test/src/test/java/org/hiero/mirror/test/e2e/acceptance/client/HookClient.java

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package org.hiero.mirror.test.e2e.acceptance.client;
+
+import com.hedera.hashgraph.sdk.AccountId;
+import com.hedera.hashgraph.sdk.EvmHookCall;
+import com.hedera.hashgraph.sdk.FungibleHookCall;
+import com.hedera.hashgraph.sdk.FungibleHookType;
+import com.hedera.hashgraph.sdk.Hbar;
+import com.hedera.hashgraph.sdk.HookEntityId;
+import com.hedera.hashgraph.sdk.HookId;
+import com.hedera.hashgraph.sdk.KeyList;
+import com.hedera.hashgraph.sdk.LambdaSStoreTransaction;
+import com.hedera.hashgraph.sdk.PrivateKey;
+import com.hedera.hashgraph.sdk.TransferTransaction;
+import jakarta.inject.Named;
+import org.hiero.mirror.test.e2e.acceptance.config.AcceptanceTestProperties;
+import org.hiero.mirror.test.e2e.acceptance.props.ExpandedAccountId;
+import org.hiero.mirror.test.e2e.acceptance.response.NetworkTransactionResponse;
+import org.springframework.retry.support.RetryTemplate;
+
+/**
+ * Client for creating and executing Hook-related transactions. This client handles hook attachment, LambdaStore
+ * transactions, and other hook storage operations.
+ */
+@Named
+public final class HookClient extends AbstractNetworkClient {
+
+    public HookClient(
+            SDKClient sdkClient, RetryTemplate retryTemplate, AcceptanceTestProperties acceptanceTestProperties) {
+        super(sdkClient, retryTemplate, acceptanceTestProperties);
+    }
+
+    public LambdaSStoreTransaction getLambdaSStoreTransaction(ExpandedAccountId account, long hookId) {
```

Comment:
Prefer we don't execute SDK transactions outside of client. Instead, let's do it similar to how we do it for `AccountClient.updateAccount()`.

```suggestion
    public NetworkTransactionResponse hookStore(ExpandedAccountId account, long hookId, Consumer<LambdaSStoreTransaction> customizer) {
```